Lib/http/server.py

```diff
@@ -332,6 +333,11 @@ def parse_request(self):
                 return False
         self.command, self.path = command, path
 
+        # bpo-43223: The purpose of replacing '//' with '/' is to provide protection 
+        # against open redirect attack reside within http.server module which can be 
+        # triggered if the path contains '//' at the beginning.
```

Comment:
Please explain that URL likes //path are treated as absolute URL with no scheme (similar to http://path), whereas /path is treated as a relative path.